src/components/tutorial-card/index.tsx

```diff
@@ -15,11 +15,7 @@ import {
 	CardBadges,
 	CardBadgeOption,
 } from 'components/tutorial-collection-cards'
-import {
-	buildAriaLabel,
-	getSpeakableDuration,
-	TutorialCardWithAuthElements,
```

Comment:
removed `TutorialCardWithAuthElements` "passthrough" export to avoid import side effects with `next-auth` that break tests

See: https://github.com/hashicorp/dev-portal/actions/runs/6015530104/job/16317581010?pr=2148#step:5:825
```
    /home/runner/work/dev-portal/dev-portal/node_modules/next-auth/src/react/index.tsx:11
    import * as React from "react"
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1728:14)
      at Object.<anonymous> (node_modules/next-auth/react/index.js:37:37)
```